backends/bmv2/CMakeLists.txt

```diff
@@ -29,7 +29,6 @@ set (BMV2_SIMPLE_SWITCH_SRCS
     simple_switch/simpleSwitch.h
     simple_switch/options.h
     )
-add_cpplint_files (${CMAKE_CURRENT_SOURCE_DIR} "${BMV2_SIMPLE_SWITCH_SRCS}")
```

Comment:
Are these still checked?

Reply:
They are checked by a [new CI run](https://github.com/p4lang/p4c/actions/runs/3406226319/jobs/5664768570) now. It should actually more thorough than what we had before because you do not have to manually add files for linting any more. 

We are adding yet another CI test, but thankfully, this one only takes about 1-2 minutes.